README.md

```diff
@@ -11,3 +11,58 @@ The data points available for this project are all going to be from GH Archive.
 updated hourly, and will provide ample data for a daily update. 
 
 The data will be delivered in a SQL Database, and updated daily. 
+<<<<<<< Updated upstream
+
+##Models 
+
+There are different types of models available to users that will display 
+different fact/dimensional information in an efficient manner. The one we 
+plan on using is a Kimball Model. 
+
+### Different Fact and Dimensions Models 
+1. Transactional Fact Model 
+This model is typically used in to relate business transactions, and 
+usually includes facts like sales orders, invoices, and shipments to 
+structure their data. 
+
+2. Snapshot Fact Model 
+Another business model, a snapchat fact model provides insight for how a 
+business entity might look at a certain point in time.
+ 
+3.Slowly Changing Dimension Model 
+This model is used to document changes to dimensions over time. Some 
+typical attributes used here are dates, historical values, and flags that 
+indicate the type of changes. 
+
+### The Kimball Model - "Star Schema" 
+
+The Kimball Model is also known as a star schema since it is shaped like a 
+star when looking at the structure of the schema. In a Kimball Model, 
+there are a couple central fact tables with multiple dimension tables 
+branching off each one. 
+
```

Comment:
Suggestion: Since we're working like a real readme, we want to tell our users what models are available to them. Let's add a section listing all of our fact and dimension models:
`fact_stars`
`fact_commits`
`dim_repositories`
`dim_users`

And list the columns in each of the models.